scripts/linux/cross-platform-rust-build.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
```

Comment:
Arnav recommended a separate script because there is a lot of edgelet specific logic in the edgelet build packaging.sh file.